docs/security-monitoring.md

```diff
@@ -23,11 +23,14 @@ Organizations using the agency add-on have the option to disable security monito
 
 ### Configuring Notifications
 Every user receives security notifications by email for all packages they have access to by default.
-Users can unsubscribe either from individual packages, or from all security notifications, if they not wish to receive any email notifications.
+Users can unsubscribe either from individual packages or from all security notifications if they do not wish to receive any email notifications.
 
-In addition to registered users receiving security notifications by email, you can configure notifications to be sent
-to other email addresses, e.g. a security mailinglist, or to a Slack channel.
-Create a notification channel for the desired destination and configure which notifications it should receive in the organization's security settings.
+You can create a notification channel for the desired destination and configure which notifications it should receive in the organization's security settings.
```

Comment:
What is a desired destination? This made sense in the previous context, but it doesn't really apply anymore? Please read this paragraph again and try and consider whether this makes any sense for a user who doesn't know about notification channels yet.

Reply:
I think this section can be removed entirely. What's worth mentioning for users is where to add a notification channel.

Reply:
I think there should still be an explanation that afterwards you have to select which packages notifications should go to which channel for.

Reply:
ok added it